app/src/main/res/values/strings.xml

```diff
@@ -185,7 +185,7 @@
     <string name="settings_user_independent">User-Independent</string>
     <string name="owner_only_summary">Only owner has root access</string>
     <string name="owner_manage_summary">Only owner can manage root access and receive request prompts</string>
-    <string name="user_indepenent_summary">Each user has his/her own separate root rules</string>
```

Comment:
This is very unelegant.

Reply:
I agree this should be "their", since otherwise it's inelegant. 😉